spec/requests/api/v1/driver_commission_histories_spec.rb

```diff
@@ -1,24 +1,64 @@
 require 'rails_helper'
 
 RSpec.describe '/api/v1/all', :type => :request do
```

Comment:
Are you talking about line 3, maybe this should be /api/v1/driver_commissions_history.